src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -17,28 +17,28 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "symfony/event-dispatcher": "~2.5.9|~2.6,>=2.6.2|~3.0.0",
-        "symfony/http-foundation": "~2.5,>=2.5.4|~3.0.0",
+        "symfony/event-dispatcher": "~2.5.9|~2.6,>=2.6.2",
+        "symfony/http-foundation": "~2.5,>=2.5.4",
         "symfony/debug": "~2.6,>=2.6.2",
```

Comment:
When adding the ~3.0.0 back to 2.8, be careful not to added them at places where they were removed from 2.7, like this one